app/utils/answersHelpers/consignment/HouseConsignmentAnswersHelper.scala

```diff
@@ -102,18 +102,32 @@ class HouseConsignmentAnswersHelper(
             helper.descriptionRow,
             helper.declarationType,
             helper.countryOfDestination,
-            helper.grossWeightRow,
-            helper.netWeightRow,
+            Seq(helper.grossWeightRow),
+            Seq(helper.netWeightRow),
             helper.cusCodeRow,
-            helper.commodityCodeRow,
-            helper.nomenclatureCodeRow,
+            Seq(helper.commodityCodeRow),
+            Seq(helper.nomenclatureCodeRow),
             helper.dangerousGoodsRows
           ).flatten,
           children = Seq(
             helper.packageSections,
             helper.documentSections,
             helper.additionalReferencesSection
-          ).flatten
+          ).flatten,
+          viewLinks = Seq(
+            itemsAddRemoveLink(itemIndex.position),
+            helper.packagingAddRemoveLink,
+            helper.documentAddRemoveLink,
+            helper.additionalReferenceAddRemoveLink
```

Comment:
Can we put a TODO here that each of these will need to be moved into their respective parent sections